app/src/main/java/com/fstyle/structure_android/screen/BaseRecyclerViewAdapter.java

```diff
@@ -0,0 +1,26 @@
+package com.fstyle.structure_android.screen;
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+import android.support.v7.widget.RecyclerView;
+
+/**
+ * Created by le.quang.dao on 14/03/2017.
+ * Base Adapter.
+ *
+ * @param <VH> is a type extend from {@link RecyclerView.ViewHolder}
+ */
+
+public abstract class BaseRecyclerViewAdapter<VH extends RecyclerView.ViewHolder>
```

Comment:
WARNING: [GenericsNaming] Generics names should be a one letter long and upper case.
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/naming.html#GenericsNaming